Logs/README.md

```diff
@@ -0,0 +1,13 @@
+# Logs
+```
+{
+  type: String,
+  timestamp: Time,
+  meta: {
+    <Fields to filter logs>
```

Comment:
In comments or in description above, please provide some example structure of <Fields to filter logs>.

Reply:
Will do so